sysutils/ctop/Portfile

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/bcicen/ctop 0.7.5 v
+revision            0
+
+categories          sysutils devel
+license             MIT
+maintainers         {outlook.de:judaew @judaew} openmaintainer
+
+description         Top-like interface for container metrics
+long_description    \
+    ctop provides a concise and condensed overview of real-time metrics for \
+    multiple containers.\nctop comes with built-in support for Docker and \
+    runC\; connectors for other container and cluster systems are planned for \
+    future releases.
+homepage            https://ctop.sh
+
+checksums           rmd160  8743fb5fd970399665c5ecc9ec885635af495e11 \
+                    sha256  b191ffa8ba343edb47644fcbc57868a282a4bd720b1836f8e0ec15820880d23d \
+                    size    1291305
+
+set date_build [clock format [clock seconds] -format %Y-%m-%d]
```

Comment:
If I am not mistaken, then the BUILD from the Makefile indicates the build date. What do you think of this way of specifying the date?